mmv1/products/dns/terraform.yaml

```diff
@@ -16,6 +16,20 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   ManagedZone: !ruby/object:Overrides::Terraform::ResourceOverride
     id_format: 'projects/{{project}}/managedZones/{{name}}'
     examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dns_domain_tutorial"
+        primary_resource_id: "default"
+        skip_docs: true
+        vars:
+          dns_compute_instance: "dns-compute-instance"
+          allow_http_traffic: "allow-http-traffic"
+          example_zone_googlecloudexample: "example-zone-googlecloudexample"
```

Comment:
variable names usually have something to do with the resource they pertain to, especially for `name` fields. This could be better read as something like `dns_managed_zone` or something